sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -981,22 +981,114 @@ func marshalWindowingStrategy(c *CoderMarshaller, w *window.WindowingStrategy) (
 	} else {
 		mergeStat = pipepb.MergeStatus_NON_MERGING
 	}
+	trigger := makeTrigger(w.Trigger)
+	accMode := makeAccumulationMode(w.AccumulationMode)
 	ws := &pipepb.WindowingStrategy{
 		WindowFn:         windowFn,
 		MergeStatus:      mergeStat,
-		AccumulationMode: pipepb.AccumulationMode_DISCARDING,
 		WindowCoderId:    windowCoderId,
-		Trigger: &pipepb.Trigger{
+		Trigger:          trigger,
+		AccumulationMode: accMode,
+		OutputTime:       pipepb.OutputTime_EARLIEST_IN_PANE,
```

Comment:
This changed from OutputTime_END_OF_WINDOW to this. We probably don't want that right now.